cmd/bom/cmd/generate.go

```diff
@@ -129,6 +130,13 @@ func (opts *generateOptions) Validate() error {
 			}
 		}
 	}
+
+	if opts.workDir != "" {
```

Comment:
How do we envision this working with multiple directory options?

For instance, suppose the user ran

```
$ bom generate -f bom/bin/bom -f kind/bin/kind -o bombom.spdx
```

Should the user be able to specify multiple `-C` options for both of those or should it be disallowed if the user specifies more than one `-f` or -`d` ?

Reply:
I think it should apply to all relative paths, WDYT?

Reply:
humm, that is more complex than this simple change here.
should we close this and rethink?

Reply:
I think with `tar` it is repeatable, but it's basically just running the equivalent of `cd` before whatever pattern? 

Given: 

```
 tree
.
├── resourcelist.yaml
└── demo-app
    ├── Chart.yaml
    ├── templates
    │   ├── NOTES.txt
    │   ├── _helpers.tpl
    │   ├── configmap.yaml
    │   ├── deployment.yaml
    │   ├── ingress.yaml
    │   ├── service.yaml
    │   └── tests
    └── values.yaml
```